sdk/sdk-view/src/test/java/mil/emp3/api/WCSTest.java

```diff
@@ -0,0 +1,75 @@
+package mil.emp3.api;
+
+
+import android.test.mock.MockContext;
+import android.util.Log;
+import android.webkit.URLUtil;
+
+import junit.framework.Assert;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.powermock.api.mockito.PowerMockito;
+
+import java.net.URL;
+import java.util.List;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+
+import mil.emp3.api.enums.KMLSEventEnum;
+import mil.emp3.api.interfaces.IKMLS;
+import mil.emp3.api.interfaces.IMapService;
+
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+/**
+ * Created by Matt.Miller on 10/12/2017.
+ */
+public class WCSTest extends TestBaseSingleMap{
```

Comment:
This pull request is the Rectangle test.  This should not be in this pull request